Files/homework/FileSharing/src/main/java/com/epam/files/App.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.files;
+
+import com.epam.files.db.FileDB;
+
+import java.nio.file.Path;
+import java.util.Scanner;
+
+public class App {
+    private final static String helpString =
+            """
+            File Sharing App
+            Usage:
+            $ gradle run --args="<address> <db name> <user> <password>"
```

Comment:
It is OK for the study application
But for real apps it would make sense to consider put datasource parameters into secure config files/services or environment variables